pkg/config/validation/validation.go

```diff
@@ -137,6 +137,11 @@ var (
 		})
 
 	validateFuncs = make(map[string]ValidateFunc)
+
+	defaultGatewayGVK = config.GroupVersionKind{
```

Comment:
can just use `gvk.KubernetesGateway`.

Notably though we don't set the version here so its not a GVK really... just a GK